.babelrc

```diff
@@ -5,7 +5,8 @@
     "transform-es2015-destructuring",
     "transform-es2015-parameters",
     "transform-async-to-generator",
-    "transform-strict-mode"
+    "transform-strict-mode",
+    ["transform-es2015-modules-commonjs", {"allowTopLevelThis": true}]
```

Comment:
I added the `allowTopLevelThis` option to the babel-plugin, otherwise [this](https://github.com/facebook/jest/blob/9b157c3a7c325c3971b2aabbe4c8ab4ce0b0c56d/packages/jest-runtime/src/__tests__/test_root/RegularModule.js#L13-L16) throws. Might not be an issue, not sure?